rasa/utils/tensorflow/layers_utils.py

```diff
@@ -0,0 +1,51 @@
+import tensorflow as tf
+from tensorflow import Tensor, TensorShape
+from typing import Union
+
+
+def random_indices(batch_size: Tensor, n: Tensor, n_max: Tensor) -> Tensor:
+    """Creates batch_size * n random indices that run from 0 to n_max.
+    
+    Args:
+        batch_size: Number of items in each batch
+        n: Number of random indices in each example
+        n_max: Maximum index
+
+    Returns:
+        A uniformly distributed integer tensor of indices
+    """
+    return tf.random.uniform(shape=(batch_size, n), maxval=n_max, dtype=tf.int32)
+
+
+def batch_flatten(x: Tensor) -> Tensor:
+    """Makes a tensor 2D.
```

Comment:
You could take what you wrote under `Returns` and move a shortened version of it here for better summary 🙂